python/ray/tests/test_advanced_4.py

```diff
@@ -185,13 +185,18 @@ def ping(self):
 def get_gcs_memory_used():
     import psutil
 
-    m = sum(
-        [
-            process.memory_info().rss
-            for process in psutil.process_iter()
-            if process.name() in ("gcs_server", "redis-server")
-        ]
-    )
+    m = 0
+    for process in psutil.process_iter():
+        try:
+            if process.name() in ("gcs_server", "redis-server"):
+                m += process.memory_info().rss
+        except psutil.NoSuchProcess:
+            # NOTE(rickyx):
```

Comment:
Do we expect to have zombie process?

Reply:
Good question - I might be wrong but if worker exited or any process exited as zombie, this will throw.

Reply:
Actually - the zombie was a "gcs_server". Hmmm, maybe I need to investigate more instead of catching and continuing.

Reply:
Not sure if @mwtian have any thoughts on this.

Reply:
Sorry missed this earlier. I don't have much context on why there can be zombie gcs ..